common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2232,7 +2232,9 @@ public static enum ConfVars {
         "If this is set to true the URI for auth will have the default location masked with DEFAULT_TABLE_LOCATION"),
     HIVE_ICEBERG_ALLOW_DATAFILES_IN_TABLE_LOCATION_ONLY("hive.iceberg.allow.datafiles.in.table.location.only", false,
         "If this is set to true, then all the data files being read should be withing the table location"),
-
+    HIVE_ICEBERG_COMPACTION_PARTITION("hive.iceberg.compaction.partition", "",
```

Comment:
why do we need HiveConf for that?

Reply:
Thanks, replaced with a constant in Context class.